frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -2,7 +2,7 @@ import { isBreakpoint, kea } from 'kea'
 import api from 'lib/api'
 import { dashboardsModel } from '~/models/dashboardsModel'
 import { router } from 'kea-router'
-import { dayjs, now } from 'lib/dayjs'
+import { Dayjs, dayjs, now } from 'lib/dayjs'
```

Comment:
`kea-typegen` gets confused because of `Dayjs` being imported for some reason… But also Day.js seems overkill for this, Date should be able to do the job too